dreamstyler/inference_style_transfer.py

```diff
@@ -82,18 +85,22 @@ def style_transfer(
     cross_attention_kwargs = {"num_stages": num_stages}
 
     content_image = Image.open(content_image_path)
+    content_image = content_image.resize((resolution, resolution))
     control_image = processor(content_image, to_pil=True)
     pos_prompt = [prompt.format(f"{placeholder_token}-T{t}") for t in range(num_stages)]
 
     outputs = []
-    for _ in range(num_samples):
+    # torch.manual_seed(1)    
```

Comment:
Is there any specific reason to change num_samples to one? if not, could you rollback this?

Reply:
Oh, I modified it for a quick experiment, but I didn't restore it. This is my mistake. I'll rollback it :D